src/quick-demo/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { RainwayRuntime } from "rainway-sdk";
+import React, { useEffect, useRef, useState } from "react";
+import ReactDOM from "react-dom";
+import "../style.css";
+import useAsyncEffect from "use-async-effect";
+
+const QuickDemo = () => {
+  const [runtime, setRuntime] = useState<RainwayRuntime | undefined>();
+  const [error, setError] = useState<string>("");
+  useAsyncEffect(async (isAlive) => {
```

Comment:
The fact that we need this third party library in the demo suggests to me that we should probably have hooks for runtime state, and just generally make `rainway-react` a bit more reactive. I imagine something like a provider component that accepts a runtime and then hooks that listen for events via context. But that's not really a problem with the demo itself.